spec/rachets/lets_not_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "spec_helper"
+
+RSpec.describe "Rachet down usage of let!" do
+  expected_usages = 698
+
+  it "ratchets down let!" do
+    command = %{grep -rn "let\!(:" spec}
```

Comment:
may want to add an `\s` in front of the regex to avoid matching methods that are like `inlet!`